core/tests/protractor/privileges.js

```diff
@@ -26,27 +26,29 @@ var workflow = require('../protractor_utils/workflow.js');
 
 describe('Permissions for private explorations', function() {
   it('should be correct for collaborators', function() {
-    users.createUser('alice@example.com', 'Alice');
-    users.createUser('bob@example.com', 'Bob');
-    users.createUser('eve@example.com', 'Eve');
+    users.createUser('user1@privileges.com', 'user1_privileges');
```

Comment:
Please keep the original alice, bob and eve here as their names fit the subsequent roles.